app/lib/efile/md/md502_calculator.rb

```diff
@@ -74,13 +74,13 @@ def calculate
         set_line(:MD502_LINE_22, :calculate_line_22)
         set_line(:MD502_LINE_22B, :calculate_line_22b)
 
+        @md502cr.calculate
```

Comment:
🤔 why did you move the calculator here as opposed to after `MD502_LINE_21`? do future calculations for 502cr depend on 22 and 22b? 

[dust] it could be helpful to add a comment which lines 502CR depends on from 502 and vice versa.

Reply:
ah, that makes sense!